src/web-ui/src/public/Main.vue

```diff
@@ -84,6 +85,14 @@ export default {
             personalized: !!personalizeRecipe,
             explanation,
           };
+
+          if (personalizeRecipe) {
+            this.userRecommendationsTitle = this.personalizeRecommendationsForVisitor
+              ? 'Inspired by your shopping trends'
```

Comment:
I think James wants us to go with "Recommended for you" rather than "Inspired by your shopping trends"